src/cobra/test/test_flux_analysis/test_loopless.py

```diff
@@ -37,14 +33,14 @@ def construct_ll_test_model():
     scope="function",
     params=[s for s in ["glpk", "cplex", "gurobi"] if s in sutil.solvers],
 )
-def ll_test_model(request):
+def ll_test_model(request: str) -> Model:
```

Comment:
`request` is not of type string (`request.param` is). I think the type is `pytest.FixtureRequest` but I'm not sure.

Reply:
Oh yes you are right, I overlooked it.